modules/migration/oaep-data-migration/src/main/java/org/wso2/migration/DataMigrationClientImpl.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.migration;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.migration.exception.DataMigrationException;
+import org.wso2.migration.service.Migrator;
+
+import java.util.List;
+
+/**
+ * Implementation of Data migration client.
+ **/
+public class DataMigrationClientImpl implements DataMigrationClient {
+    private static final Log LOG = LogFactory.getLog(DataMigrationClientImpl.class);
```

Comment:
The logger reference is not a constant, but a final reference, and should not be in uppercase. Check all places

Reply:
Fixed by 325baaba7cbaefc9ed303543ab90a17e859fcc1a